irc/client.go

```diff
@@ -1122,7 +1121,8 @@ var (
 func (session *Session) SendRawMessage(message ircmsg.IrcMessage, blocking bool) error {
 	// use dumb hack to force the last param to be a trailing param if required
 	var usedTrailingHack bool
-	if commandsThatMustUseTrailing[message.Command] && len(message.Params) > 0 {
+	config := session.client.server.Config()
+	if config.Server.Compatibility.forceTrailing && commandsThatMustUseTrailing[message.Command] && len(message.Params) > 0 {
```

Comment:
`forceTrailing` or `ForceTrailing`?

Reply:
`forceTrailing` is correct. This is the pattern where the capitalized member is the one directly deserialized from the YAML, and then it gets postprocessed into a usable value. In this case, `ForceTrailing` is a `*bool` (allowing us to detect whether it is unset), and then we convert "unset" into `true` during the load process when we set `forceTrailing`.

Reply:
Ah, I see. In future I'll probably reverse that so the lowercase ones are the internal members nobody needs to touch (e.g. `motdName`) and the exposed ones are all uppercased. But sounds good with our current pattern.